examples/src/bin/sleep_timer.rs

```diff
@@ -1,36 +0,0 @@
-//! Demonstrates deep sleep with timer wakeup
```

Comment:
Can we move the sleep examples to qa test? I don't think we'll be able to HIL test it (maybe it might work), but they're useful to do some manual testing with.

Reply:
Sorry, could you move the other sleep ones too?

Reply:
> Can we move the sleep examples to qa test?

Oh, example**S** 😆  Will be done in no time